### PR TITLE
Allows you to choose between three lipstick styles

### PR DESCRIPTION
**Pull request by vinylspiders**
## About The Pull Request

Somehow the lipstick sprite got messed up and was a pixel lower than it should have been, ending up on the chin instead of the lips.

Edit: I just found https://github.com/tgstation/tgstation/pull/55281 in the history and now I'm not sure anymore...

I've instead opted into making three different lipstick styles with a radial to select which one you want to put on your face. You can change styles via alt click. This way if you want to have it lower like Queen Amidala you can, or higher if you're like me and think that looks better.

dreamseeker_rAdQi13WZ2

dreamseeker_h2kUUGppud

<details><summary>Lower (Aka how is is in the current build)</summary>

dreamseeker_ak13Ou69ER

</details>

<details><summary>Upper (All pixels shifted up one)</summary>

dreamseeker_eeMXvE9y4z

</details>

<details><summary>Middle (The new default- pixels shifted up one for the profile sprites)</summary>

dreamseeker_FCBJyoAalS

</details>

## Why It's Good For The Game

More character customization options

## Changelog

:cl:
image: you can now change the style of lipstick to be higher or lower on the face by alt-clicking the lipstick tube
/:cl:

### CODE REVIEW

**Review by Wallemations — APPROVED** [review]
I See no harm in it

File: code/game/objects/items/cosmetics.dm [hunk 1]
```diff
@@ -22,6 +28,11 @@
 	if(vname == NAMEOF(src, open))
 		update_appearance(UPDATE_ICON)
 
+/obj/item/lipstick/examine(mob/user)
+	. = ..()
+
+	. += "Alt-click on [src] to change the style."
```
Comment (MrMelbert) [inline code/game/objects/items/cosmetics.dm:34]:
```suggestion
	. += "Alt-click to change the style."
```
"This is red lipstick. Alt click on the red lipstick to change the style."
vs
"This is red lipstick. Alt click to change the style."

File: code/game/objects/items/cosmetics.dm [hunk 2]
```diff
@@ -35,6 +46,41 @@
...
+	if(!pick)
+		return TRUE
+
+	switch(pick)
+		if(MIDDLE_LIP)
+			style = "lipstick"
+		if(LOWER_LIP)
+			style = "lipstick_lower"
+		if(UPPER_LIP)
+			style = "lipstick_upper"
```
Comment (MrMelbert) [inline code/game/objects/items/cosmetics.dm:75]: `return TRUE` at the end?

File: code/game/objects/items/cosmetics.dm [hunk 3]
```diff
@@ -35,6 +45,42 @@
 	colored_overlay.color = lipstick_color
 	. += colored_overlay
 
+/obj/item/lipstick/AltClick(mob/user)
+	. = ..()
+	if(.)
+		return
```
Comment (san7890) [inline code/game/objects/items/cosmetics.dm:51]:
```suggestion
		return TRUE
```

this is already implicitly implied so let's just make it explicitly stated

**Review by san7890 — CHANGES_REQUESTED** [review]

**Pull request merged**